src/components/editor/EditAuthorsForm.jsx

```diff
@@ -84,8 +41,8 @@ export default class EditAuthorsForm extends BaseComponent {
               <div key={author.slug}>
                 {
                   !striked
-                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleDeleteAuthor.bind(this, author.slug, true)}>&times;&nbsp;</a>
-                    : <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleUnstrikeAuthor.bind(this, author.slug)}>~&nbsp;</a>
+                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.props.handleDeleteAuthor.bind(null, author.slug, true)}>&times;&nbsp;</a>
```

Comment:
And you mean the last true? It's a parameter passed to the handleDeleteAuthor function that specifies whether the author being deleted is an original author, aka, should it be just striked, or deleted from addedAuthors